fleetshard/pkg/runtime/runtime.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/stackrox/acs-fleet-manager/fleetshard/pkg/util"
 	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/api/private"
 	"github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager"
+	fmImpl "github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager/impl"
```

Comment:
since there is just one occurence of `fleetmanager` in this file I think we can swap this imports.
```suggestion
	fm "github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager"
	fleetmanager "github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager/impl"
```